jd-gui.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+/app/jre/bin/java -jar /app/jd-gui.jar $1
```

Comment:
```suggestion
exec /app/jre/bin/java -jar /app/jd-gui.jar $1
```

Reply:
Why?

Reply:
so that there is no uneeded shell process left over. That's how wrapper script are written.